opencti-platform/opencti-front/src/private/components/common/cards/GenericAttackCard.tsx

```diff
@@ -145,6 +160,7 @@ GenericAttackCardProps
       addBookmark(cardData.id, entityType);
     }
   };
+  const image = cardData.images?.edges ?? [];
```

Comment:
`images` in plural since it's an array.
Also, maybe it could be useful to have directly the avatarImage here, like this : 

```
const images = cardData.images?.edges ?? [];
const avatarImage = (images.length > 0 && images[0]) ? images[0].node : null;

```
And then you could directly use avatarImage, and only test if it's defined or not. What do you think ?